### PR TITLE
[FSDE-281] Add disable property for row to be unselectable

### PR DESCRIPTION
**Pull request by aryaroudi**
hey @janibolkvadze please review this!

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://c44b935de490a6baaaa215ce843c098208aea837-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/2/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/3/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://c17d2a08be008d9d5f5e27ba0baeb731e10394df-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/4/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/5/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://545682ed316e765b215a60a39892b015eb8cf27e-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/6/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/7/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://3d0f9f63ecaf50f227fe9ef2ed41f261ecd3b252-chi-pr.enterprise.ctl.io

File: src/chi-vue/src/constants/types.ts [hunk 1]
```diff
@@ -45,6 +45,7 @@ export interface DataTableRow {
   rowId: string;
   rowNumber: string;
   selected?: boolean;
+  disabledSelection?: boolean;
```
Comment (janibolkvadze) [inline src/chi-vue/src/constants/types.ts:48]:
```suggestion
  selectionDisabled?: boolean;
```
Thoughts on calling it the other way around?

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/8/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/9/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://55e9531ace855b6b7864d6150f4ecc1bcfcefa70-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/10/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/11/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://5ea6ec98204dd767d462818b7701a5971282da40-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/12/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/13/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/14/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/15/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/16/. :x:

**Review by janibolkvadze — APPROVED** [review]

**Review by jllr — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/17/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-107/18/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://7131ebaa05d3089bb5590d47d0093110aa1e38ab-chi-pr.enterprise.ctl.io

**Pull request merged**